src/content/cloud/registry.mdx

```diff
@@ -50,3 +50,25 @@ Any image pushed into the Okteto Registry is private. You'll need to authenticat
 Namespaces in Okteto Cloud are automatically allowed to pull images that belong to their namespace automatically. If your application uses images from the Okteto Registry, it'll be able to pull container images without any extra configuration.
 
 To pull an image from a different namespace, you'll need to create and configure the required `imagePullSecrets` as outlined [here](/docs/reference/faqs/#how-to-use-private-images).
+
+## Push helm chart into the Okteto Registry
+
+Just as we can push images to the Okteto Registry, you also have the option to push helm charts. To do this, the first step is to login to the registry where we want to push the chart:
+
+```
+helm registry login -u <user> <registry-domain>
```

Comment:
Can we add a specific example as wall on the expected value of <registry-domain> like we do on the "Push images into the Okteto Registry" section?
```suggestion
helm registry login -u <user> <registry-domain>
```

Also, I don't see a mention of the password. Do they need to create a key or something?

Reply:
The password is requested by helm once the helm registry login is executed. I will update the image to include it and make it more transparent.